### DISCUSSION TITLE
[QUESTION] Problems with installing ptb

### DISCUSSION THREAD

**Discussion by Kotov584** [python-telegram-bot/python-telegram-bot#3257] [Q&A]
### Steps to Reproduce

1. pip install python-telegram-bot --pre
2. python bot.py 

Fresh install doesn't work anywhere  
Tried to install telegram with pip but result is the same

### Expected behaviour

Expected it work

### Actual behaviour

Traceback (most recent call last):
  File "main.py", line 1, in <module>
    from telegram import Update
ModuleNotFoundError: No module named 'telegram'

### Operating System

Windows 10 / Linux

### Version of Python, python-telegram-bot & dependencies

```shell
Python 3.8.2
python-telegram-bot last one
```

**Comment by Poolitzer** [comment]
@Kotov584 I will go out on a limb and say that <code>pip</code> does not link to the same python version as <code>python</code> does. Try <code>python -m pip install python-telegram-bot </code>;P

**Comment by Kotov584** [comment]
Same result

**Comment by Kotov584** [comment]
Now it runs with python (python3 doesn't works) bot.py but I dont have any responses from bot lol

**Reply by Poolitzer** [reply]
enable verbose logging, that should give you an idea where it might be hanging
